pkg/sdk/logging/model/output/zz_generated.deepcopy.go

```diff
@@ -1403,6 +1403,16 @@ func (in *SyslogOutputConfig) DeepCopyInto(out *SyslogOutputConfig) {
 		*out = new(secret.Secret)
 		(*in).DeepCopyInto(*out)
 	}
+	if in.PrivateKeyPath != nil {
```

Comment:
you added 5 new fields, but this file has only two, is it correct?

Reply:
This file is autogenerated using `make generate`  command